geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/ClientHealthMonitorJUnitTest.java

```diff
@@ -145,53 +189,55 @@ private int createServer() {
    */
   @Test
   public void testDeadClientRemovalByServer() throws Exception {
+    System.setProperty(ClientHealthMonitor.CLIENT_HEALTH_MONITOR_INTERVAL_PROPERTY,
```

Comment:
This test has many outstanding issues that I considered outside the scope of this change. I'll add a ticket for reworking it. Most importantly this test should not make a cache and just unit test the class it claims to be testing. This test takes > 1.2s whereas the other examples that I added run in ~1mms